packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -47,7 +47,7 @@ export class Autosuggest {
    * the original object that was used to create the suggestion.
    */
   @Prop()
-  readonly suggestions: Suggestion[] = [];
+  readonly suggestions: Suggestion[] | null = null;
```

Comment:
Ik denk dat het fraai is om ook de documentatie van deze property uit te breiden.